library.json

```diff
@@ -7,37 +7,24 @@
   "platforms": ["espressif8266", "espressif32"],
   "authors":
   {
-    "name": "PhracturedBlue",
-    "url": "https://github.com/PhracturedBlue"
+    "name": "simone1999/PhracturedBlue",
+    "url": "https://github.com/simone1999"
   },
   "repository":
   {
     "type": "git",
-    "url": "https://github.com/PhracturedBlue/ESP8266MQTTMesh.git"
+    "url": "https://github.com/simone1999/ESP8266MQTTMesh.git"
   },
   "dependencies": [
     {
       "name": "ESPAsyncTCP",
-      "version": "^1.1.3",
+      "version": "1.1.3",
       "platforms": "espressif8266"
     },
     {
       "name": "AsyncMqttClient",
-      "version": "^0.8.1",
+      "version": "0.8.1",
       "platforms": "espressif8266"
-    },
-    {
```

Comment:
Is this no longer needed for ESP32 support?

Reply:
I think they are needed, I reverted to previous version. Thanks.